packages/typespec-azure-resource-manager/src/resource.ts

```diff
@@ -126,26 +127,31 @@ export interface ResourceMetadata {
 }
 
 /** New details for a resolved resource */
-export interface ResolvedResource extends ResourceMetadata {
+export interface ResourceModel extends ResourceModelMetadata {
   /** The set of resolved operations for a resource.  For most 
         resources there will be 1 returned record */
-  operations?: ResolvedOperations[];
+  resources?: ResolvedResource[];
```

Comment:
I am not sure if `resources` could be accurate here, because last time when I tried to call this API on employee from the resource manager sample spec, I get two items here, one corresponding for CRUD of that ARM resource, and the other is the list by subscription.
Would `resourceOperations` be better here?